README.md

```diff
@@ -50,7 +50,9 @@ _Resulting layout:_
 ```html
 <header>
   <h1>Blog 2000</h1>
-  <h2>All articles</h2>
+  <yield to="header">
```

Comment:
We can refactor this later. Maybe it should be `<pop in="header">` for all we know, reads good ^^